src/App.js

```diff
@@ -1,24 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Great Job adding an App component, However please kindly make sure that the app component is be class-based